plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -165,6 +231,49 @@ def disruption_budget_target(deployment)
       end
     end
 
+    def add_env_config_map
+      return unless env_from_config_map?
+      return if env_config_map_exists?
```

Comment:
maybe i'm just not a rubyist enough but this reads better to me as a generalist...

`return if env_config_map_exists? || !env_from_config_map?`